sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/RowToPubsubMessage.java

```diff
@@ -40,19 +39,14 @@
  * whether config.getValue("timestampAttributeKey") is set.
  */
 class RowToPubsubMessage extends PTransform<PCollection<Row>, PCollection<PubsubMessage>> {
-  private final Row config;
   private final Boolean useTimestampAttribute;
 
-  private RowToPubsubMessage(Row config, Boolean useFlatSchema, Boolean useTimestampAttribute) {
-    checkArgument(useFlatSchema, "RowToPubsubMessage is only supported for flattened schemas.");
-
-    this.config = config;
+  private RowToPubsubMessage(Boolean useTimestampAttribute) {
     this.useTimestampAttribute = useTimestampAttribute;
   }
 
-  public static RowToPubsubMessage fromConfig(
-      Row config, Boolean useFlatSchema, Boolean useTimestampAttribute) {
-    return new RowToPubsubMessage(config, useFlatSchema, useTimestampAttribute);
+  public static RowToPubsubMessage fromConfig(Boolean useTimestampAttribute) {
```

Comment:
Nit: Now that `config` parameter is gone, this class could be better named as something more meaningful like `withTimestampAttribute()`.

Reply:
Done